src/linear_algebra/basis.lean

```diff
@@ -751,6 +751,63 @@ end
 
 end span
 
+lemma smul_group_span_eq_top
```

Comment:
Since the group is on the LHS of the smul, I think this should be `group_smul`, same for smul throughout